test/distributed/test_c10d_gloo.py

```diff
@@ -181,7 +181,7 @@ class RendezvousEnvTest(TestCase):
     @retry_on_connect_failures
     def test_logging_init(self):
         os.environ["WORLD_SIZE"] = "1"
-        os.environ["MASTER_ADDR"] = "127.0.0.1"
+        os.environ["MASTER_ADDR"] = "::1"
```

Comment:
This (and the following) test will likely fail on an IPv4-only machine. Another option would be to use `localhost` instead of an IP address.

Reply:
Yes, I've fixed those when I noticed the failures (switched to `localhost`)